src/main/resources/application-dev.yml

```diff
@@ -16,5 +16,5 @@ spring:
   data:
     cassandra:
       local-datacenter: datacenter1
-  profiles:
-    include: debug
\ No newline at end of file
+#  profiles:
```

Comment:
Need to remove this commented line as spring.profiles.debug is not supported in new spring boot version.